kokoro/ubuntu/release.sh

```diff
@@ -59,27 +59,21 @@ if [ "$SOURCE_VERSION" != "$TAG_VERSION" ]
         exit 1 # terminate and indicate error
 fi
 
-echo "Installing aktau/github-release.."
+echo "Installing github-release"
 go get github.com/itchio/gothub
 cd github/google-cloud-intellij
 echo "Building plugins"
 ./gradlew buildPlugin
 
 echo "Creating Github release for tag: $GIT_TAG_NAME"
-
```

Comment:
we want to keep release.sh as is. This is actually a newer version than the branch.